core/src/main/java/io/aiven/klaw/service/TopicSyncControllerService.java

```diff
@@ -817,10 +831,10 @@ public ApiResponse updateSyncTopicsBulk(SyncTopicsBulk syncTopicsBulk) throws Kl
       }
     } else {
       try {
-        List<TopicConfig> topicsMap =
+        LoadTopicsResponse loadTopicsResponse =
             getTopicsFromKafkaCluster(
-                syncTopicsBulk.getSourceEnv(), syncTopicsBulk.getTopicSearchFilter());
-        for (TopicConfig hashMap : topicsMap) {
+                syncTopicsBulk.getSourceEnv(), syncTopicsBulk.getTopicSearchFilter(), false);
+        for (TopicConfig hashMap : loadTopicsResponse.getTopicConfigSet()) {
```

Comment:
Nitpick, it says hashMap, but this is a topic config from a set so can probably improve the name to avoid confusion